client/src/App.js

```diff
@@ -1,22 +1,26 @@
-import logo from './logo.svg';
+//import logo from './logo.svg';
```

Comment:
minor PR formatting thing, I try to remove unused comments before merging (unless I plan on using them in the very near future) - but up to you since this isn't a 'live' app yet :)

Reply:
I will definitely remove this until it's needed, thank you!